dll/win32/iernonce/iernonce.h

```diff
@@ -7,4 +7,22 @@
 
 #pragma once
 
-typedef VOID(CALLBACK *RUNONCEEX_CALLBACK)(UINT CompleteCnt, UINT TotalCnt, DWORD Reserved);
+#define WIN32_NO_STATUS
+
+#include <windows.h>
+#include <winbase.h>
+#include <windef.h>
+#include <windowsx.h>
+#include <shlwapi.h>
+#include <winreg.h>
+#include <process.h>
```

Comment:
What is this header for?

Reply:
`<windowsx.h>` is for useful macros like `SelectFont` and so on
`<shlwapi.h>` is for `PathGetArgsW` and `PathRemoveArgsW`
`<process.h>` is for `_beginthreadex`

`<winreg.h>` seems already covered by Windows.h, I will remove it!

Reply:
Don't use the whole windows.h.

Reply:
I've already moved `<process.h>` to where it belongs. so, what should be put inside pch, and what shouldn't 🤣